src/fal/cli/model_generator/model_generator.py

```diff
@@ -64,6 +64,12 @@ def _generate_python_dbt_models(model_path: Path):
         with open(sql_path, "w") as file:
             file.write(sql_contents)
 
+        if not old_checksum or checksum != old_checksum:
+            logger.warn(
+                f"File {sql_relative_path} generated from {py_relative_path}.\n"
```

Comment:
Should we quote this (just like the warning below)?
```suggestion
                f"File '{sql_relative_path}' was generated from '{py_relative_path}'.\n"
```